src/main/java/org/opentripplanner/model/StopTime.java

```diff
@@ -309,29 +293,16 @@ public void cancel() {
     dropOffType = PickDrop.CANCELLED;
   }
 
-  public void cancelDropOff() {
-    dropOffType = PickDrop.CANCELLED;
-  }
-
-  public void cancelPickup() {
-    pickupType = PickDrop.CANCELLED;
-  }
-
   @Override
   public String toString() {
-    return (
-      "StopTime(seq=" +
-      getStopSequence() +
-      " stop=" +
-      getStop().getId() +
-      " trip=" +
-      getTrip().getId() +
-      " times=" +
-      TimeUtils.timeToStrLong(getArrivalTime()) +
-      "-" +
-      TimeUtils.timeToStrLong(getDepartureTime()) +
-      ")"
-    );
+    return ToStringBuilder
```

Comment:
Revert this. StopTime is a ValueObject and we make short toStrings() for them. These objects might be dumped in hundreds in logs and test(may be not this one). One of the reasons is to avoid DOS'ing logs. Also, VOs are always part of something, therefore logging referenced objects is a potensial cyclic call (trip and stop).

Reply:
I did it because the `toString` threw an NPE during a test. I solved it by setting the proper values: bb38a6c3d62ebcf7b64db26a93dd9f48967a3404